ghcide/src/Development/IDE/Plugin/CodeAction/ExactPrint.hs

```diff
@@ -367,17 +374,28 @@ extendImportTopLevel thing (L l it@ImportDecl{..})
       then lift (Left $ thing <> " already imported")
       else do
 #if !MIN_VERSION_ghc(9,2,0)
-        when hasSibling $
-          addTrailingCommaT (last lies)
+        anns <- getAnnsT
+        maybe (pure ()) addTrailingCommaT (lastMaybe lies)
         addSimpleAnnT x (DP (0, if hasSibling then 1 else 0)) []
         addSimpleAnnT rdr dp00 [(G AnnVal, dp00)]
+
+        -- When the last item already has a trailing comma, we append a trailing comma to the new item.
+        let isAnnComma (G AnnComma, _) = True
+            isAnnComma _                  = False
```

Comment:
nitpick: can we align this or remove the superfuous space?

Reply:
Aha, my mistake. btw, stylish-haskell refuses to run on this file, because there are too many CPP, making it fail.